shared/src/main/scala/coop/rchain/catscontrib/MonadError_.scala

```diff
@@ -0,0 +1,28 @@
+package coop.rchain.catscontrib
+import cats.MonadError
+
+trait MonadError_[F[_], E] extends ApplicativeError_[F, E] {
```

Comment:
It would make sense to test laws of this type class

Reply:
See my comment down below - I'll apply our resolution from that discussion here as well